tools/amend-event-data.js

```diff
@@ -543,7 +561,12 @@ function applyEventPatches(spec) {
           continue;
         } else if (patch.change) {
           for (const target of Object.keys(patch.change)) {
-            event[target] = patch.change[target];
+            if (patch.change[target]) {
```

Comment:
```suggestion
            if (patch.change[target] !== null) {
```
Shouldn't this check specifically for `null` equality? otherwise this would have an expected result e.g. on `change: {bubbles: false}`